src/goto-programs/string_instrumentation.cpp

```diff
@@ -281,7 +282,7 @@ void string_instrumentationt::do_sprintf(
   {
     error().source_location=target->source_location;
     error() << "sprintf expected to have two or more arguments" << eom;
-    throw 0;
+    UNREACHABLE;
```

Comment:
The check as a whole should be an invariant (several occurrences).

Reply:
This is not an invariant either; it is an input condition check. An invariant is something that the program maintains internally, and not something it expects the environment to provide.

Reply:
I would expect this to be ensured by the C frontend (which it actually does). After that we should be able to assume it invariant.